### PR TITLE
Add standing delegations document

### PR DESCRIPTION
**Pull request by pradyunsg**
This also documents the current standing delegations to PyPA.

/cc @ncoghlan

### CODE REVIEW

**Comment by ncoghlan** [comment]
Just noting that this PR came out of a conversation I had with @pradyunsg at PyCon US, where I mentioned that we'd updated PEP 1 to explicitly cover the concept of standing delegations, but I hadn't yet followed up with a concrete proposal to create an explicit Steering Council record of the PyPA delegations.

File: standing-delegations.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Standing Delegations
+
+This document serves as a public record of current standing delegations, the
+reasons they were put in place, and the circumstances under which they may no
+longer be needed.
+
+## PyPA Specifications
```
Comment (gvanrossum) [inline standing-delegations.md]: I would have expected this section heading to read "PyPA Delegations".

**Comment by warsaw** [comment]
Questions:

* Wouldn't this be better in the devguide?  It seems a bit weird to put it in the toplevel of the CPython repo (what about alternative implementations? :)
* Why md format when all of our other documentation is reST?

**Comment by gvanrossum** [comment]
This is not the CPython repo, it's the Steering Council repo. :-)

**Comment by warsaw** [comment]
> This is not the CPython repo, it's the Steering Council repo. :-)

Oops. nm :)

**Comment by ncoghlan** [comment]
Approved at Steering Council meeting on the 14th/15th of May

- PyPA section title updated
- moved into process subfolder

**Pull request merged**